src/delagent/ui/admin-upload-delete.php

```diff
@@ -114,27 +115,79 @@ function Delete($uploadpk, $Depends = NULL)
   } // Delete()
 
   /**
+   * @param $uploadpks
+   * @brief starts deletion and handles error messages
+   * @return string
+   */
+  function initDeletion($uploadpks)
+  {
+    $V = "";
+    $errorMessages = [];
+    $deleteResponse = "";
+    for($i=0; $i < sizeof($uploadpks); $i++)
+    {
+      $deleteResponse = $this->TryToDelete($uploadpks[$i]);
+
+      if($deleteResponse->getDeleteMessageCode() != DeleteMessages::SUCCESS)
+      {
+        $errorMessages[] = $deleteResponse;
+      }
+    }
+
+    /**
+     * Output normal message, if only one upload
+     */
+    if(sizeof($uploadpks) < 2)
+    {
+      $V .= DisplayMessage($deleteResponse->getDeleteMessageString().$deleteResponse->getAdditionalMessage());
+    }
+    else
+    {
+      $displayMessage = "";
+
+      if(in_array(DeleteMessages::SCHEDULING_FAILED, $errorMessages))
+      {
+        $displayMessage .= "<br/>Scheduling failed for " .
+          array_count_values($errorMessages)[DeleteMessages::SCHEDULING_FAILED] . " uploads<br/>";
+      }
+
+      if(in_array(DeleteMessages::NO_PERMISSION, $errorMessages))
+      {
+        $displayMessage .= "No permission to delete " .
+          array_count_values($errorMessages)[DeleteMessages::NO_PERMISSION]. " uploads<br/>";
+      }
+
+      $displayMessage .= _("Deletion of ") .
+        (sizeof($uploadpks)-sizeof($errorMessages)) . " projects queued";
```

Comment:
here you use `gettext` inconsistently. You add it for the first half of the sentence but not the second.